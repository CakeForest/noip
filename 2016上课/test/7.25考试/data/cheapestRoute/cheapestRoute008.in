20
22 43 67 0 79 39 34 79 23 82 79 2 86 83 80 10 31 75 33 48 72 -1 51 
50 13 5 12 9 3 12 16 4 2 14 20 18 6 14 18 1 11 17 6 2 20 5 16 8 18 9 11 4 20 2 12 20 4 6 9 1 18 4 14 12 4 12 21 10 4 12 15 11 20 14 
50 13 12 5 0 17 18 11 7 12 6 20 2 0 14 11 16 1 6 4 12 13 16 3 4 11 5 19 12 11 6 4 15 3 3 18 5 18 20 1 7 16 6 10 2 5 13 19 21 2 6 
160
10 19 91 60 58 -1 14 18 -1 36 26 
50 7 2 7 3 4 6 5 8 7 6 6 7 1 7 9 6 0 9 6 8 8 1 2 8 1 3 4 8 8 6 8 2 5 6 2 6 7 5 3 0 0 0 0 1 7 2 5 2 7 8 
50 4 3 6 1 0 6 4 9 5 4 8 3 3 2 3 8 7 1 7 6 4 9 5 8 5 9 8 6 3 0 7 2 5 8 6 6 1 2 9 9 8 6 9 1 4 1 6 5 6 3 
36
46 26 49 -1 65 91 61 38 98 25 38 42 20 6 52 85 86 50 19 42 42 80 54 87 98 10 33 27 2 75 -1 29 44 58 7 12 9 89 29 51 8 48 47 57 43 17 68 
34 2 14 25 24 19 29 20 4 4 43 32 27 40 37 26 29 31 7 38 30 45 13 38 27 43 40 14 43 4 30 21 26 37 2 
34 32 31 37 38 8 35 41 7 30 35 12 4 30 43 43 8 41 0 27 8 14 33 37 33 40 2 1 44 19 37 42 30 38 5 
60
3 58 37 58 
21 0 2 1 0 2 0 2 0 0 2 2 0 0 0 1 1 1 1 0 0 2 
21 2 2 2 1 2 1 1 0 1 2 0 0 1 2 2 1 0 2 1 0 1 
144
23 86 18 42 80 85 46 48 48 -1 83 80 64 13 63 17 33 25 83 98 17 53 36 71 
31 14 20 0 13 2 8 15 21 8 16 8 10 12 19 10 9 17 20 19 16 5 4 5 1 22 12 8 17 6 1 13 
31 10 9 5 2 19 8 8 3 14 2 6 14 18 1 11 22 5 6 2 0 14 14 6 15 9 8 3 17 6 17 13 
54
20 77 69 91 47 87 30 40 87 99 52 84 8 75 37 3 79 47 31 49 63 
33 9 11 15 4 17 12 10 7 1 9 15 16 19 4 10 16 19 9 2 16 16 14 0 14 12 6 4 3 7 8 2 19 0 
33 1 13 18 3 9 10 0 10 6 17 3 15 7 0 0 6 12 0 11 7 17 15 19 6 16 10 10 13 3 0 16 13 14 
30
41 81 78 77 76 84 99 15 72 97 18 37 2 97 16 0 57 6 25 96 95 56 37 75 6 91 2 91 63 1 100 17 78 79 82 7 11 16 95 24 11 63 
42 28 14 0 12 40 28 19 10 34 37 32 25 21 19 0 38 12 18 20 37 1 25 3 35 10 4 13 20 14 34 2 8 12 32 40 23 3 8 34 9 39 12 
42 18 0 38 8 23 24 8 11 35 10 15 17 11 13 27 31 37 36 35 2 9 33 35 38 36 38 38 39 2 8 33 34 28 22 39 26 15 29 28 30 28 14 
180
27 56 64 -1 20 39 56 -1 79 -1 38 85 -1 -1 27 93 52 19 51 61 55 30 90 -1 -1 1 95 35 
50 9 5 26 8 7 4 6 20 19 3 24 5 5 17 1 15 2 26 18 10 7 22 12 12 24 17 7 16 6 8 17 20 14 21 14 3 15 25 10 13 14 13 24 21 8 18 7 21 15 4 
50 0 21 9 12 1 6 2 0 17 1 0 1 22 4 8 19 23 8 12 2 21 8 3 16 5 10 10 6 22 20 22 18 20 8 9 5 23 22 11 22 21 21 14 12 8 12 2 1 0 23 
150
36 25 59 -1 61 26 12 52 6 28 73 98 -1 100 83 11 26 63 43 70 91 16 40 14 30 37 94 6 3 15 43 67 72 100 24 7 -1 
39 12 8 8 21 16 22 21 2 13 25 13 25 8 7 5 14 0 8 0 31 22 14 2 0 6 12 8 4 25 23 25 21 9 20 3 34 19 32 6 
39 0 35 4 0 11 9 5 26 11 6 34 9 31 33 3 33 4 19 18 22 24 25 32 16 17 20 15 0 9 17 21 7 24 10 7 31 4 20 7 
0
50 39 -1 71 44 73 70 17 -1 74 100 58 20 56 76 -1 39 38 76 42 55 31 74 18 95 11 94 64 63 46 66 7 -1 51 -1 20 95 88 -1 59 -1 53 48 7 12 55 63 -1 26 -1 100 
10 12 15 16 46 27 26 41 14 23 30 
10 20 33 42 15 2 49 7 27 29 38 
100
6 71 63 52 -1 37 17 
35 3 3 3 3 5 4 5 4 4 3 5 0 5 5 2 2 2 3 0 5 4 0 5 0 0 1 4 4 0 2 4 1 0 4 3 
35 5 3 1 5 1 0 5 3 5 3 2 1 5 1 5 0 5 0 3 4 2 5 2 5 4 2 2 5 4 1 5 5 5 2 2 
324
38 60 45 85 49 52 28 39 -1 59 6 1 98 7 83 51 84 -1 62 52 62 71 0 -1 57 14 63 44 42 -1 98 95 47 33 66 1 74 25 81 
49 20 27 33 17 16 14 17 15 28 22 29 10 11 16 31 24 3 15 29 31 5 35 26 8 16 36 35 14 28 7 12 31 33 4 35 30 34 21 21 9 37 32 4 8 11 19 21 33 19 
49 15 21 37 18 15 35 3 9 9 9 28 22 33 30 22 1 2 7 7 15 3 21 25 16 19 25 15 1 13 18 7 13 32 18 31 31 26 0 15 4 12 22 35 14 1 16 7 28 36 
0
44 7 42 -1 99 36 77 94 21 24 77 27 89 83 31 29 21 45 35 62 63 46 24 59 27 99 69 99 35 92 7 64 20 80 23 53 2 -1 87 96 15 4 7 52 1 
24 21 4 27 28 6 39 43 39 4 2 30 0 36 27 2 16 16 30 41 29 5 17 23 15 
24 43 21 18 30 3 3 0 19 3 19 34 30 8 32 22 23 5 19 42 39 3 0 11 21 
270
13 73 -1 -1 -1 -1 -1 -1 -1 -1 -1 -1 -1 -1 
1 0 
1 8 
0
22 33 -1 70 29 52 24 85 2 79 100 4 76 44 16 38 8 87 52 30 63 26 37 
44 17 21 14 1 3 11 14 6 2 15 2 10 11 12 5 18 10 15 4 13 18 17 15 20 9 9 10 18 10 15 15 0 3 21 20 19 2 1 7 2 9 7 15 11 
44 17 1 5 12 16 13 21 15 5 15 5 20 15 17 3 5 15 9 2 19 16 20 2 18 5 17 2 9 0 13 17 2 14 14 7 16 5 0 20 17 0 1 21 7 
18
6 66 89 -1 28 92 59 
8 4 3 5 0 5 1 0 0 
8 1 2 5 4 5 3 3 1 
54
45 75 74 100 36 71 23 -1 55 73 95 67 97 78 10 57 84 32 83 98 37 -1 79 12 48 97 -1 64 18 -1 25 -1 19 35 7 14 19 20 65 70 2 59 11 72 61 98 
24 1 33 1 1 33 34 11 14 26 0 13 20 15 25 1 13 29 30 26 22 34 21 6 27 
24 7 2 27 39 41 37 40 15 6 42 35 1 21 9 35 35 5 38 11 13 44 32 16 1 
96
39 85 21 68 2 38 36 89 33 4 68 -1 12 91 89 93 10 0 71 28 19 50 74 64 48 23 4 32 40 93 45 59 36 47 39 64 57 46 72 55 
36 37 35 16 29 37 31 1 4 16 9 5 18 10 30 6 8 26 1 28 6 34 14 6 16 23 24 2 5 34 31 12 20 27 21 34 8 
36 8 30 8 5 34 6 8 17 38 30 35 6 3 32 1 22 38 26 38 0 7 27 5 26 14 32 1 2 32 26 14 7 25 6 7 2 
400
30 5 61 61 36 32 21 5 73 67 82 82 98 4 54 80 79 86 62 39 97 68 -1 49 5 -1 62 77 33 -1 24 
21 10 11 5 26 12 0 4 28 10 18 6 8 29 13 20 1 1 13 9 0 6 
21 29 4 25 20 16 21 20 18 0 11 29 26 11 1 12 3 10 22 5 26 11 
300
13 96 9 95 65 65 37 84 70 88 61 81 -1 23 
43 0 0 1 7 3 10 6 3 2 6 12 3 6 3 5 12 1 11 0 5 9 11 5 10 6 2 5 4 1 3 2 1 1 3 6 5 3 0 6 10 9 4 6 
43 12 10 2 6 1 3 9 12 11 10 7 10 4 12 10 5 5 0 7 3 3 7 11 6 4 2 10 8 10 0 10 10 12 10 1 12 12 4 6 4 10 1 3 
0
