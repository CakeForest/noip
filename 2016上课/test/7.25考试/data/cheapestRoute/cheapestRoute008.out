474 5 
36 1 
-1  -1
95 2 
274 6 
30 1 
254 3 
-1  -1
-1  -1
-1  -1
324 1 
299 8 
612 7 
-1  -1
86 4 
113 2 
255 3 
738 9 
516 7 
0 1 
